data/STRchive-database.json

```diff
@@ -219,9 +219,9 @@
         "Mechanism":"Polyglutamine",
         "Mechanism_detail":"Polyglutamine",
         "Mechanism_source":"(doi.org\/10.1038\/nrg.2017.115) (doi.org\/10.1007\/s11604-022-01343-5)",
-        "source":"Hannan 2018, Mirkin 2007, GeneReviews NBK535148, s40478-021-01201-x",
-        "notes":"Can have 'CAA' as an interruption (PMID: 35245110)",
-        "details":null,
+        "source":"Hannan 2018, Mirkin 2007, GeneReviews NBK535148, s40478-021-01201-x, doi.org/10.1016/j.neurol.2017.03.019, doi.org/10.1016/j.pneurobio.2012.05.007, doi.org/10.1111/odi.12121, DOI: 10.1007/s00415-018-8968-7, DOI: 10.1007/s12031-015-0684-5",
+        "notes":"Possibility that contractions may play a role in disease (PMID: 10398229). May be subclinical in females (PMID: 34922802). Interruptions appear not to play a role (PMID: 24041967). Can be clinically heterogeneous even within the same family (PMID: 20184516); may present with clinical heterogeneity.",
```

Comment:
What does "Interruptions appear not to play a role" mean? That they don't exist, or that they do occur, but don't change the pathogenicity of the allele?